nginx/templates/cache-blacklist.conf.tmpl

```diff
@@ -0,0 +1,22 @@
+{{ $params := (datasource "cachingIgnoreParams").params -}}
+{{ if $params }}
+  {{ range  $params -}}
+    if ($c_uri ~ (.*)(?:&|^){{ . }}=[^&]*(.*)) { set $c_uri $1$2 ; }
+  {{ end -}}
+
+  # remove dangling &
+  if ($c_uri ~ ^&(.*)) {
+    set $c_uri $1;
+  }
+
+  set $c_uri $is_args$c_uri;
+
+  # remove lonely ?
+  if ($c_uri ~ ^\?$) {
+      set $c_uri "";
+  }
+
+  # finally we have stripped out parameters and have nice cache key
```

Comment:
Did you get inspired by a blog post or stackoverflow question for this? If yes, It might be good to add it here as a comment. 😊

Reply:
Yes, heavily _inspired_.